Rnn

AbdelrahmanRadwan opened the pull request:
Added two classes,
class ProjectionOp:
to create Single layer perceptron algorithm implementation.

class Model:
to create and build the model

AbdelrahmanRadwan committed:
Start the branch

AbdelrahmanRadwan committed:
Update Neural Network.py

ibrahimsharaf commented on Neural Network.py:
```diff
@@ -0,0 +1,124 @@
+            self.dtype)
+
+
+    def create_rnn_cell():
```
Please use docstrings to explain the function role.

AbdelrahmanRadwan committed:
Merge the work, start fixing Sharaf notes, Start adding the Neural Cell function usage, and step function

AbdelrahmanRadwan committed:
Merge the work

AbdelrahmanRadwan committed:
Added the step and create_cell functions

Added some functions in the neural network class.
Did some modifications to the rest of classes.
